pages/docs/tutorials/websocket/websocket-request-reply.md

```diff
@@ -0,0 +1,369 @@
+        payload:
+          type: object
+          description: 'Payload of the reaction added event'
+          properties:
+            token:
+              type: string
+            team_id:
+              type: string
+            event:
+              type: object
+              properties:
+                user:
+                  type: string
+                  description: User ID who performed this event
+                reaction:
+                  type: string
+                  description: The only reaction that you need is a heart emoji
+                item_user:
+                  type: string
+                  description: |
+                    User ID that created the original item that has been reacted
+                    to
+                item:
+                  type: object
+                  properties:
+                    channel:
+                      type: string
+                      description: Channel information of original message
+                    ts:
+                      type: string
+                      description: Timestamp information of original message
+                event_ts:
+                  type: string
+                  description: Reaction timestamp
+        type:
+          type: string
+        accepts_response_payload:
+          type: boolean `}
+</CodeBlock>
+
+Additionally, you will also be adding the `acknowledge` schema that makes use of the `envelope_id` attribute to send a reply back to Slack acknowledging that event has been received.
+
+<CodeBlock language="yaml" highlightedLines={[4,5,6]}>
+{`    acknowledge:
+      type: object
+      properties:
+        envelope_id:
+          type: string
+          description: 'Unique ID of acknowledged payload'
+        payload:
+          type: object
+          description: 'Optional payload of event' `}
+</CodeBlock>
+
+## Define acknowledge message to channel
+You can extend the channel used by  `Heart-Counter` to now include the `acknowledge` message.
+
+<CodeBlock language="yaml" highlightedLines={[9,10]}>
+{`channels:
+  root:
+    address: /
+    messages:
+      hello:
+        $ref: '#/components/messages/hello'
+      reaction:
+        $ref: '#/components/messages/reaction'
+      acknowledge:
+        $ref: '#/components/messages/acknowledge' `}
+</CodeBlock>
+
+## Define operations
+
+That brings to the important part of the tutorial where you get to finally represent the request reply pattern. 
+
+Both `helloListener` and `reactionListener` operations are set to `receive` events. However, in the case of `reactionListener` you also want to represent the message that is sent back to the server. This is where the `reply` attribute comes into play. 
+
+Since both the request and reply function happens over the same WebSocket URL, both the `channel` values stay the same. However, you can differentiate the message each operation conveys by specifying the message it will send or receive. 
```

Comment:
```suggestion
Since both the request and reply function happens over the same WebSocket URL, both `channel` values stay the same. However, you can differentiate each operation's message by specifying the messages sent or received. 
```